www/src/routes/volunteer/ResumeReview.tsx

```diff
@@ -51,7 +52,17 @@ const ResumeReview: FC = () => {
         <TitledPage title='Resume Review'>
             <Grid container justify='center' alignItems='center' direction='column'>
                 <Grid container justify='flex-start' alignItems='flex-start' direction='column'>
-                    <Typography variant='h2'>Currently Reviewing</Typography>
+                    <Grid container>
+                        <Grid item xs={9}>
+                            <Typography variant='h2'>Currently Reviewing</Typography>
+                        </Grid>
+                        <Grid item xs={3} container justify='flex-end'>
+                            <Button onClick={() => dispatch(refreshResumeReviews())} variant='contained' color='primary' startIcon={<RefreshIcon />}>
```

Comment:
Can you disable this with `availableIsLoading || reviewingIsLoading` just to prevent double fetching